taxi/tests/test_forms.py

```diff
@@ -0,0 +1,146 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from taxi.models import Car, Manufacturer
+from taxi.forms import (
+    CarForm,
+    DriverCreationForm,
+    DriverLicenseUpdateForm,
+    DriverSearchForm,
+    CarSearchForm,
+    ManufacturerSearchForm,
+    validate_license_number
+)
+from django.core.exceptions import ValidationError
+
+
+class CarFormTest(TestCase):
+    def setUp(self):
+        self.driver1 = get_user_model().objects.create_user(
+            username="driver1",
+            password="testpass123",
+            first_name="John",
+            last_name="Doe",
+            license_number="ABC12345"
+        )
+        self.driver2 = get_user_model().objects.create_user(
+            username="driver2",
+            password="testpass456",
+            first_name="Jane",
+            last_name="Smith",
+            license_number="XYZ67890"
+        )
+        self.manufacturer = Manufacturer.objects.create(
+            name="Toyota",
+            country="Japan"
+        )
+        self.car = Car.objects.create(
+            model="Corolla",
+            manufacturer=self.manufacturer
+        )
+
+    def test_car_form_valid(self):
+        form_data = {
+            "model": "Camry",
+            "manufacturer": self.manufacturer.pk,
+            "drivers": [self.driver1.pk, self.driver2.pk]
+        }
+        form = CarForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_car_form_invalid(self):
+        form_data = {
+            "model": "",
+            "manufacturer": "",
+            "drivers": []
+        }
+        form = CarForm(data=form_data)
+        self.assertFalse(form.is_valid())
+
+
+class DriverCreationFormTest(TestCase):
+    def test_driver_creation_form_valid(self):
+        form_data = {
+            "username": "newdriver",
+            "password1": "testpass123",
+            "password2": "testpass123",
+            "license_number": "ABC12345",
+            "first_name": "New",
+            "last_name": "Driver"
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_driver_creation_form_invalid_license(self):
+        form_data = {
+            "username": "newdriver",
+            "password1": "testpass123",
+            "password2": "testpass123",
+            "license_number": "ABC12",
+            "first_name": "New",
+            "last_name": "Driver"
```

Comment:
The creation of driver1 and driver2 in the setUp method of CarFormTest is repetitive.

The form_data dictionaries in test_car_form_valid and test_car_form_invalid methods of CarFormTest and in both test methods of DriverCreationFormTest are repetitive.